src/app/containers/SocialEmbed/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useContext } from 'react';
+import path from 'ramda/src/path';
+import styled from 'styled-components';
+import {
+  AmpSocialEmbed,
+  CanonicalSocialEmbed,
+} from '@bbc/psammead-social-embed';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import useToggle from '#hooks/useToggle';
+import socialEmbedBlockPropTypes from '#models/propTypes/socialEmbed';
+import createTranslations from './translations';
+
+const MAX_WIDTH = '31.25rem';
```

Comment:
Where does this `MAX_WIDTH` value derive from?

Reply:
Good question – I've added a comment that explains this.

Reply:
We use `rem` units across our codebase, so that our UI scales if users change their font-size. Using pixel values here would mean this wouldn't scale.